InvenTree/plugin/helpers.py

```diff
@@ -116,25 +117,26 @@ def get_git_log(path):
     from InvenTree.ready import isInTestMode
 
     output = None
-    path = path.replace(str(settings.BASE_DIR.parent), '')[1:]
+    path = os.path.abspath(path)
+
+    if os.path.exists(path) and os.path.isfile(path):
+        path = os.path.dirname(path)
 
     # only do this if we are not in test mode
     if not isInTestMode():  # pragma: no cover
 
         try:
-            walker = Repo.discover(path).get_walker(paths=[path.encode()], max_entries=1)
-            try:
-                commit = next(iter(walker)).commit
```

Comment:
I think this is a regression.
The current implementation highlights the last change to the particular plugin, not the repo as a whole. This is why we use a `walker` that seeks a change in the particular file. Your implementation just shows the latest commit to the repo. You can see this with the build modules - they now all show the latest change to the whole repo, not the particular plugin.

Reply:
Surely we want to see the commit hash for the `HEAD` of the git repo, right? The exact state of the particular file can be directly inferred from that.

Additionally each individual plugin reports its own version.

To be clear, what you are saying is that given the "current implementation", a git repo containing *multiple* plugins would potentially report a *different* git hash for each plugin, even though they are all running from the same `HEAD` commit?

Reply:
The current implementation is to look at each plugin. You can nest plugins (like we do in the samples folder) and you would want the individual change dates and authors to show up, not the head. At that point, the whole git integration becomes a lot less useful.

Reply:
So what you would want to see is the latest commit which touched any file associated with a particular plugin? I can see how this could introduce difficulties.